wrappers/node/test/suite1/ariesvcx-issuer-credential.test.ts

```diff
@@ -18,54 +18,16 @@ describe('IssuerCredential:', () => {
     });
 
     it('throws: missing sourceId', async () => {
-      const { sourceId, ...data } = await dataIssuerCredentialCreate();
-      const error = await shouldThrow(() => IssuerCredential.create(data as any));
-      assert.equal(error.vcxCode, VCXCode.INVALID_OPTION);
-    });
-
-    it('throws: invalid credDefHandle', async () => {
-      const { credDefHandle, ...data } = await dataIssuerCredentialCreate();
-      const error = await shouldThrow(() => IssuerCredential.create(data as any));
-      assert.equal(error.vcxCode, VCXCode.INVALID_CREDENTIAL_DEF_HANDLE);
-    });
```

Comment:
Can add similar test for the send offer. Do we want to maintain nodejs unit tests?